src/core/libmaven/groupFiltering.cpp

```diff
@@ -40,7 +40,12 @@ bool GroupFiltering::filter(PeakGroup &peakgroup)
 
     peakgroup.groupStatistics();
 
-    if (_mavenParameters->clsf->hasModel() && peakgroup.goodPeakCount < _mavenParameters->minGoodGroupCount)
+    if (_mavenParameters->clsf->hasModel()) {
+        _mavenParameters->clsf->classify(&peakgroup);
+        peakgroup.updateQuality();
+    }
+
+    if (peakgroup.goodPeakCount < _mavenParameters->minGoodGroupCount)
```

Comment:
If there is no model file, all groups will be filtered out. This condition should be moved to the previous `if` block

Reply:
Fixed